### PR TITLE
Fixup build with qt6

### PR DESCRIPTION
**Pull request by neochapay**

### CODE REVIEW

File: src/lib/wall-settings.cpp [hunk 1]
```diff
@@ -152,7 +152,11 @@ QString Maemo::Timed::WallClock::wall_settings_pimple_t::str() const
   os << "time_at_zero=" << time_at_zero.str().c_str() << ", " ;
   os << "offset=" << offset << ", " ;
   os << "zone='" << zone << "', " ;
+#if (QT_VERSION < QT_VERSION_CHECK(6,0,0))
```
Comment (pvuorela) [inline src/lib/wall-settings.cpp:155]: 5.14

File: src/lib/wall-info.cpp [hunk 2]
```diff
@@ -211,7 +211,11 @@ QString Maemo::Timed::WallClock::Info::str() const
   os << " clock_back=" << p->systime_diff ;
   os << " nitz_supported=" << p->nitz_supported ;
   os << " default_timezone=" << p->default_timezone.toStdString().c_str() ;
+#if (QT_VERSION < QT_VERSION_CHECK(6,0,0))
```
Comment (pvuorela) [inline src/lib/wall-info.cpp:214]: 5.14

File: tools/timedclient/timedclient.cpp [hunk 3]
```diff
@@ -1414,7 +1414,7 @@ static QList<uint> do_search(char *args)
       fprintf(stderr, "missing attribute key or value: %s=%s\n", key, val);
       continue;
     }
-    attributes.insert(key, val);
+    attributes.insert(key, QString(val));
```
Comment (pvuorela) [inline tools/timedclient/timedclient.cpp:1417]: Suppose it works but these char* conversions can warn or even be unavailable, and at least are vague on the encoding. Guess QString::fromUtf8() could be slightly better here.

**Review by pvuorela — COMMENTED** [review]
Could here also consider first the qt 5.14 api using version.

**Comment by neochapay** [comment]
@pvuorela ping

**Pull request closed without merging**